contracts/MemeBase.sol

```diff
@@ -49,9 +49,9 @@ contract MemeBase is MemeFactory {
         require(accounts.length == amounts.length);
 
         // Initiate meme token map values
-        // summonTime is set to zero such that no one is able to heart this token
+        // unleashTime is set to 1 such that no one is able to heart this token
```

Comment:
Should add a test that also no-one can purge it. I think no-one can because the token address doesn't exist yet.

Reply:
This is now possible as nonces start from 0. Need to change nonces to start from 1.